.github/CONTRIBUTING.md

```diff
@@ -24,18 +24,18 @@ If you have difficulty, ask for help in the #coding-general channel on our disco
 
```

Comment:
We'll replace this markdown file with ours in future so that change isn't needed *for now of course*